### DISCUSSION TITLE
?? in tensorflow predictions

### DISCUSSION THREAD

**Discussion by puraminy** [google-research/text-to-text-transfer-transformer#801] [Q&A]
I train `t5` transformer

Here is a sample (input, output):

input:

    b'[atomic]:<subject>PersonX plays a ___ in the war</subject><relation>oReact</relation>'

output:

    <object>none</object>

However, for the prediction I get:

     ⁇ object>none ⁇ /object>

which replaces `<` with ` ?? `, what should I do to resolve this problem?

**Comment by craffel** [comment]
< is out-of-vocabulary for the T5 tokenizer, so it gets tokenized/detokenized to ??.

**Reply by puraminy** [reply]
thank you!

wish you also said how I could add this char or change the vocabulary

I also wonder how such a common character is out of T5 tokenizer
